### PR TITLE
Color Game Timer Added / CSS on Homepage and Mem Game Improved

### PR DESCRIPTION
**Pull request by Jaketay23**
Need to hook up to backend

### CODE REVIEW

File: src/games/ColorGame.js [hunk 1]
```diff
@@ -0,0 +1,499 @@
...
+		GameOver: Symbol("Game Over")
+	});
+
+	const [startTime, setStartTime] = useState(0);
+    const [endTime, setEndTime] = useState(0);
+    const [timeDiffList, setTimeDiffList] = useState([]);
+
+
+
+	const questions = [
```
Comment (kudlakk) [inline src/games/ColorGame.js]: nitpick, but this should array of objects should either be moved to its own file in the `constants/` directory or *at least* moved outside the scope of the ColorGame component. as-is this entire object is redefined each time the component re-renders. Though performance hasn't been an issue for us so far, this has the potential to hinder performance.

File: src/games/ColorGame.js [hunk 2]
```diff
@@ -0,0 +1,499 @@
+import React, { useState, useEffect } from 'react';
+import "../css/ColorGame.css";
+import { recordData } from "./GameDataRecorder";
+import { GAMES_ENUM } from "../constants/GamesConstants";
+
+
+
+export default function ColorGame({ advanceStateFunction }) {
+    const colors_value = [
```
Comment (kudlakk) [inline src/games/ColorGame.js]: this constant, colors_names, and GameState should all also be moved outside the scope of the ColorGame component (by this I just mean put them on line 6)

File: src/games/ColorGame.js [hunk 3]
```diff
@@ -0,0 +1,499 @@
...
+	const [showScore, setShowScore] = useState(false);
+	const [score, setScore] = useState(0);
+	const [initialMinute,initialSeconds] = [0,max];
+    const [minutes, setMinutes ] = useState(initialMinute);
+    const [seconds, setSeconds ] =  useState(initialSeconds);
+	const [start, setStart] = useState(0);
+	const [count, setCount] = useState(seconds);
+	var timer;
+	
+	const GAME_TYPE = 4;
```
Comment (kudlakk) [inline src/games/ColorGame.js]: prefer to pull this from here: https://github.com/san0014s/Cog-Web-App/blob/main/src/constants/GamesConstants.js#L1

**Review by kudlakk — CHANGES_REQUESTED** [review]
Couple small things that I noticed just reading thru, not super pressing. Haven't pulled down to test on my local yet. If u don't wanna go out of ur way to make these changes, lmk and I can do it later today

File: src/games/ColorGame.js (hunk 3 above)
Comment (Jaketay23) [reply src/games/ColorGame.js]: It is. I just had that there from earlier. I just removed the GAME_TYPE const from the code. I can push it here in a second

File: src/games/ColorGame.js (hunk 1 above)
Comment (Jaketay23) [reply src/games/ColorGame.js]: I think I fixed both this and the next one in my latest commit. Moved this one outside of the loop and the other one was not being used.

**Comment by karlscheib** [comment]
So I found an oddity, not sure if it is just a me problem, but the color game, the "color game" button on the home page, and the pregame cards are a little off the page with no way of scrolling down to it. 


offscreen



offscreen2




offscreen3

**Comment by kudlakk** [comment]
can't see lives

Screen Shot 2022-11-20 at 9 25 07 AM

**Review by kudlakk — APPROVED** [review]

**Pull request merged**